src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static Boolean isRunning = true;
+
+    public static void printGreetings() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+    }
+
+    public static String findTaskSpecifics(String details) {
+        return details.substring(details.indexOf(" "), details.indexOf("/"));
+    }
+
+    public static void printDefaultTaskResponse(int number, Task[] tasks) {
+        System.out.println("Got it. I've added this task:\n" + tasks[number].toString() + "\n"
+                + "Now you have " + (number + 1) + " tasks in the list.");
+    }
+
+    public static void runTaskList() {
+        Task[] tasks = new Task[100];
+        String line;
+        int number_of_tasks = 0;
```

Comment:
The code violates the coding standard. Please consider changing the variable to camelCase to comply with coding standards.